src/types.jl

```diff
@@ -89,6 +89,9 @@ function FixedTimeZone(s::AbstractString)
     return FixedTimeZone(name, offset)
 end
 
+name(tz::FixedTimeZone) = tz.name
```

Comment:
`Base.nameof` maybe?

Reply:
Looks like `nameof` always returns a `Symbol`. I'll note I'm not exporting this accessor.

Reply:
Ah okay then

Reply:
Would you prefer if we left this accessor out completely for now?

Reply:
As long as it's not exported, it seems okay to me.